docs/src/browsers.md

```diff
@@ -0,0 +1,95 @@
+---
+id: browsers
+title: "Browsers"
+---
+
+Each version of Playwright needs specific versions of browser binaries to operate. By default Playwright downloads Chromium, WebKit and Firefox browser versions that it supports. With every release, Playwright updates the versions
+of the browsers it supports, so that the latest Playwright would support the latest browsers at any moment.
+
+<!-- TOC -->
+
+## Chromium
+
+Playwright's Chromium version matches the latest
```

Comment:
This section sounds scary for someone who is not familiar with the browser channels (and differences between them) and just wants to use the latest one (aka playwright today) so I'd rephrase it to say that by default it will test against the latest dev version of chrome and put the rest of the explanation elsewhere.

Reply:
rewrote it.